### PR TITLE
feat: add sepolia subgraphs

### PR DESCRIPTION
**Pull request by fionnachan**

### CODE REVIEW

File: packages/subgraph-common/config/nova.json
Comment (fionnachan) [inline packages/subgraph-common/config/nova.json:1]: fixed the block numbers by checking the contracts on explorers

File: packages/arb-bridge-eth/src/utils.ts [hunk 1]
```diff
@@ -34,8 +34,8 @@ export const getL2ChainId = (): Bytes => {
     }
   }
 
-  if (network == "rinkeby") return Bytes.fromByteArray(Bytes.fromHexString("0x066EEB"));
   if (network == "goerli") return Bytes.fromByteArray(Bytes.fromHexString("0x066eed"));
+  if (network == "sepolia") return Bytes.fromByteArray(Bytes.fromHexString("0x66eee"));
```
Comment (gvladika) [inline packages/arb-bridge-eth/src/utils.ts:38]: `0x66eee` -> `0x066eee` should resolve the error that arb-bridge-eth-sepolia subgraph is throwing

Comment (fionnachan) [reply packages/arb-bridge-eth/src/utils.ts]: it is syncing!! thank you

**Review by gvladika — APPROVED** [review]
LGTM

**Pull request merged**